lib/main/utils/app_config.dart

```diff
@@ -6,6 +6,7 @@ import 'package:tmail_ui_user/features/login/data/network/config/oidc_constant.d
 
 class AppConfig {
   static const int limitCharToStartSearch = 3;
+  static const int maximumMegabytesSizeFileAttachedInComposer = 10;
```

Comment:
```suggestion
  static const int warningAttachmentFileSizeInMegabytes = 10;
```

Reply:
> 

IMO, should be `warningAttachmentMaximumFileSizeInMegabytes`

Reply:
done